package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "FoodiGo-Picture-Component",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "react-dev": "webpack -d --watch",
+    "server-dev": "nodemon server/index.js",
+    "seed-data": "node database-mongo/index.js",
+    "test": "jest",
+    "coverage": "jest --coverage"
+  },
+  "author": "Matt Mullen",
+  "license": "ISC",
+  "devDependencies": {
```

Comment:
nearly all of your babel and webpack dependencies are going to be regular dependencies, not dev